### ISSUE TITLE
Cannot run due to Rocket.toml not mapped properly 

### ISSUE THREAD

**Issue by simonlaw0711** [chimbosonic/hagrid-container#1]
```(office) root@hx-hk-in-key-2:~/hagrid-docker/data$ make run
docker run --rm -it -p 7272:7272\
	-v /root/hagrid-docker/data:/var/lib/hagrid\
	-e port=7272\
	-e base_URI="https://localhost:7272"\
	-e from="noreply@localhost"\
	-e x_accel_redirect=false\
	-e token_secret="hagrid"\
	-e token_validity=3600\
	-e mail_rate_limit=60 hagrid:latest
Hagrid folder not populate generating config
thread 'main' panicked at 'Rocket config must succeed: missing field `keys_internal_dir`', src/main.rs:41:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
make: *** [Makefile:2: run] Error 101
```

**Comment by simonlaw0711** [comment]
The `Rocket.toml` file has been populate on my '/root/hagrid-docker/data' but seems still couldn't be mapped

**Comment by simonlaw0711** [comment]
```
(office) root@hx-hk-in-key-2:~/hagrid-docker/data$ ls -la
total 44
drwxr-xr-x 8 root root  4096 Aug  2 06:54 .
drwxr-xr-x 9  502 staff 4096 Aug  2 07:15 ..
-rw-r--r-- 1 root root   295 Aug  2 06:16 Makefile
-rw-r--r-- 1 root root   455 Aug  2 06:54 Rocket.toml
-rw-r--r-- 1 root root   475 Aug  2 04:17 Rocket.toml.old
drwxr-xr-x 5 root root  4096 Aug  1 14:50 assets
drwxr-xr-x 2 root root  4096 Aug  1 14:50 email-templates
drwxr-xr-x 2 root root  4096 Aug  1 14:50 errors-static
drwxr-xr-x 2 root root  4096 Aug  2 05:42 keys
drwxr-xr-x 3 root root  4096 Aug  2 05:43 public
drwxr-xr-x 7 root root  4096 Aug  1 14:50 templates
(office) root@hx-hk-in-key-2:~/hagrid-docker/data$ 
```

**Comment by chimbosonic** [comment]
Hi @simonlaw0711,
I'll try and debug and fix but going to be honest I haven't used Hagrid in a while.

**Comment by simonlaw0711** [comment]
Much appreciate!!!🙏🙏🙏

**Comment by chimbosonic** [comment]
Ok so the mapping in Rocket.toml changed and it now looks for `[release]` rather than `[production]`. I would recommend getting the Rocket.toml file from https://gitlab.com/hagrid-keyserver/hagrid/-/blob/master/Rocket.toml.dist and modify that.

I'll make changes to the build process to use that one by default and remove some of the templating stuff as I don't think its useful.

**Comment by chimbosonic** [comment]
Fixed by c3f5202ef81bb39ace4c97e152167b81d6a1f474
